fastpath/fastpath/core.py

```diff
@@ -914,27 +914,24 @@ def score_vanilla_tor(msm: dict) -> dict:
     tk = g_or(msm, "test_keys", {})
     scores = init_scores()
 
-    nks = ("error", "success", "tor_log", "tor_progress_summary", "tor_progress_tag")
+    nks = ("error", "success", "tor_logs", "tor_progress_summary", "tor_progress_tag")
     if msm["software_name"] == "ooniprobe" and all_keys_none(tk, nks):
         if tk["tor_progress"] == 0:
             # client bug?
             scores["msg"] = "Client bug"
             return scores
 
-    tor_log = tk.get("tor_log")
-    if tor_log is None:
+    tor_logs = tk.get("tor_logs") or tk.get("tor_log")
+    if tor_logs is None:
         # unknown bug
-        return scores
-
-    if (
-        "Bootstrapped 100%: Done" in tor_log
-        or "Bootstrapped 100% (done): Done" in tor_log
```

Comment:
I believe this was in place to workaround a bug where older versions of probe did not have the `tor_log` key.

Also you need to handle both values of `tor_logs` and `tor_log`, since older versions of OONI Probe had a different name for that key, see: https://github.com/ooni/spec/blob/master/nettests/ts-016-vanilla-tor.md#incompatibility-with-ooniprobe-legacy

Reply:
Updated!